other/huffmancoding/huffmancoding_test.go

```diff
@@ -0,0 +1,53 @@
+// huffmancoding_test.go
+// description: Tests the compression, encoding and decoding algorithms of huffmancoding.go.
+// author(s) [pedromsrocha](https://github.com/pedromsrocha)
+// see huffmancoding.go
+
+package huffman
+
+import (
+	"sort"
+	"testing"
+)
+
+// SymbolCountOrd computes sorted symbol-frequency list of input message
+func SymbolCountOrd(message string) ByFreq {
+	runeCount := make(map[rune]int)
+	for _, s := range message {
+		runeCount[s]++
+	}
+	listfreq := make(ByFreq, len(runeCount))
+	i := 0
+	for s, n := range runeCount {
+		listfreq[i] = SymbolFreq{symbol: s, freq: n}
+		i++
+	}
+	sort.Sort(listfreq)
```

Comment:
Forgot to mention. Could you try to use the sorting algorithm that is inside this repository - we have tried very much to avoid the standard library sort package because I think if there is something that our sort functions are missing then it makes sense to adapt them to improve this repo 😄